src/app/BigQuestion.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class BigQuestion extends Model
+{
+
+    use SoftDeletes;
+
+    protected $guarded = array('id');
```

Comment:
array()の書き方より `['id']` の短縮形が望ましいため `['id']` としてください
```suggestion
    protected $guarded = ['id'];
```